static/src/inline-svgs/journalism/audio-player/download.svg

```diff
@@ -0,0 +1,12 @@
+<svg width="25px" height="25px" viewBox="0 0 30 30" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
How are you optimising these svgs? @zeftilldeath recommends to use this tool:https://jakearchibald.github.io/svgomg/

Ideally the width/height/viewbox should be like: `width="25px" height="25px" viewBox="0 0 25 25"`. It is just easier to style that way.

Reply:
They do contains live fonts that render well only on Guardian designers machines too :-) Ask for them to be outlined, unless @regiskuckaertz tell you a way to use live fonts from within an SVG.